src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -50,6 +50,21 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
   // verify that the user is not reviewing their own change request
   if (reviewerId === foundCR.submitterId) return res.status(401).json({ message: 'Access Denied' });
 
+  // if Scope CR, make sure that a proposed solution is selected before approving
+  if (
+    foundCR.type === 'ISSUE' ||
```

Comment:
a better way to check this is seeing `if (foundCR.scopeChangeRequest) { ... }`

however, to do this, you'll need to make sure prisma includes the scope cr on line 46